merkle/merkle_test.go

```diff
@@ -0,0 +1,44 @@
+package merkle
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"context"
+	"strconv"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/trace"
+	"github.com/ava-labs/avalanchego/x/merkledb"
+)
+
+var res_root ids.ID
+var res_db merkledb.MerkleDB
+var res_err error
+
+func BenchmarkMerkleTxRoot(b *testing.B) {
+	for _, size := range []int{10, 100, 1000, 10000} {
+    ctx := context.TODO()
+    tracer := trace.Noop
+    merkleItems := make([][]byte, 0, size)
+    for i := 0; i < size; i++ {
```

Comment:
I'd recommend creating this prior to starting each benchmark.

I'd also recommend using crypto/rand to populate 32 bytes rather than this integer approach.